packages/deepsignal/core/src/index.ts

```diff
@@ -117,7 +117,7 @@ const objectHandlers = {
 	},
 	ownKeys(target: object): (string | symbol)[] {
 		if (!objToIterable.has(target)) objToIterable.set(target, signal(0));
-		objToIterable.get(target).value = objToIterable.get(target).value;
+		(objToIterable as any)._ = objToIterable.get(target).value;
```

Comment:
That assignment seems weird to me. 🤔 Do you know if it is required? Maybe you can get rid of the assignment and just read the value, am I right?

```suggestion
		objToIterable.get(target).value;
```

Reply:
Without an assignment, the minifier deletes that line. More context in this PR:

- https://github.com/luisherranz/deepsignal/pull/28

At first, I assigned it to itself and that did the trick (and it's fine because signals don't trigger if their value doesn't change), but that is captured as a side-effect on computeds. So this new assignment is just a fake one that goes nowhere and it's small when minified (4 chars long: `a._=`).